client/src/App.css

```diff
@@ -13,6 +13,22 @@
     font-family: "NexaDemo-Light";
     src: url("./assets/NexaDemo-Light.otf") format("opentype");
 }
+
+@font-face {
+    font-family: "NexaDemo-Bold";
+    src: url("./assets/NexaDemo-Bold.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "NexaTextDemo-Light";
+    src: url("./assets/NexaTextDemo-Light.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "NexaTextDemo-Bold";
+    src: url("./assets/NexaTextDemo-Bold.otf") format("opentype");
+}
+
```

Comment:
please add import for RobotoSlab-Regular.ttf
and don't forget to place the file into assets folder as well :)